database/migrations/2020_06_24_021847_create_artworks_table.php

```diff
@@ -22,7 +22,7 @@ public function up()
             $table->unsignedDecimal('length', 6, 2);
             $table->unsignedDecimal('depth', 6, 2)->nullable();
             $table->string('sku')->unique();
-            $table->unsignedDecimal('price', 8, 2);
+            $table->unsignedInteger('price_in_cents')->nullable();
```

Comment:
since the artworks are unique pieces, I think we should add an identifier that lets us know if it has been added to an order or not: this will help avoid the problem of having an already purchased artwork added to another order and also will be used to validate if the site should show the informative flyer for the artwork on display or tell the customer that it is no longer available.

Reply:
we can create a relationship with the order entity and use this to validate any race condition that could appear with this requirement